examples/rendering/demo.py

```diff
@@ -154,7 +154,7 @@ def main():
 
     ########################## forward + backward twice ##########################
     scene.reset()
-    horizon = 2000
+    horizon = 10
```

Comment:
Still not acceptable. A demo is supposed to be a demo. Not a flash on the screen. If you want to reduce duration for the CI. Use the appropriate flag.